.dockerignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+vendor
+storage/framework/cache/**
+storage/framework/sessions/**
+storage/framework/views/**
```

Comment:
Zouden hier ook de storage public/agendaItem en public/newsItems  bij moeten staan want die staan ook in onze gitignore?

Reply:
ik kan ze erbij zetten maar ze zijn in principe leeg in de repo dus worden ook niet mee gekopieerd